src/explore-education-statistics-common/src/modules/charts/types/chart.ts

```diff
@@ -72,13 +72,30 @@ export type AxesConfiguration = {
   [key in AxisType]?: AxisConfiguration;
 };
 
-export type DataClassification = 'EqualIntervals' | 'Quantiles' | 'Custom';
+export const dataGroupingTypes = {
+  EqualIntervals: 'Equal intervals',
+  Quantiles: 'Quantiles',
+  Custom: 'Custom',
+} as const;
+
+export type DataGroupingType = keyof typeof dataGroupingTypes;
 
 export interface CustomDataGroup {
   max: number;
   min: number;
 }
 
+export interface DataGroupingConfiguration {
+  type: DataGroupingType;
+  numberOfGroups: number;
+  customGroups: CustomDataGroup[];
+}
+
+export interface DataGrouping extends DataGroupingConfiguration {
+  dataSet: DataSet;
+  unit?: string;
```

Comment:
Seems like a bit of a potential bug to have this here as we don't actually want to store `unit` in the backend and we only really need this for the chart builder UI components. Consequently, this is a bit leaky and implies the wrong things to consumers of this type.

Would suggest removing this and either:

- Refactoring the type into one of the components and export it from there e.g. `DataGroupingWithUnit`
- Refactor the components to accept a `unit` prop. Looks like we already tried to do this (see `ChartDataGroupingsForm`) so it'd just be a continuation of that.

This means that in cases where we're currently doing `OmitStrict<DataGrouping, 'dataSet'>`, we can just replace this with `DataGroupingConfiguration`